module-2/lab-df-calculation-and-transformation/your-code/challenge-2.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
- Use loc for Sub Problem 3 - Collecting Information and show the head/tail of the resulting dataframe.
- Q2 and Q3 outputs don't work properly for distinct customers.